api/cloudprotocol/unitstatus.go

```diff
@@ -130,7 +131,7 @@ type NodeInfo struct {
 
 // ServiceStatus service status.
 type ServiceStatus struct {
-	ServiceID string     `json:"serviceId"`
```

Comment:
Do not change core name, you can change json tag only.

Reply:
okay